sdk/core/azure-core/src/main/java/com/azure/core/util/UserAgentUtil.java

```diff
@@ -59,10 +62,10 @@ public static String toUserAgentString(String applicationId, String sdkName, Str
 
         // Add the required default User-Agent string.
         userAgentBuilder.append(DEFAULT_USER_AGENT_HEADER)
-            .append("-")
-            .append(sdkName)
-            .append("/")
-            .append(sdkVersion);
+                .append("-")
```

Comment:
nit: I've been seeing that some PRs use 4 spaces on indentation and others 8 spaces, we should get this standardized to reduce git noise

Reply:
Yeah, it seems like the `.editorconfig` we have isn't fully configured to standardize the code style